scenario28/resources/user.properties

```diff
@@ -46,10 +46,10 @@ Googlescope=scope=openid email profile
 #Twittr_IDP
 #TwitterIdentityProviderName=Sol28GoogleIDP
 #TwitterHomeRelam=twitter
-TwitterClientId=yDNon5Ppg6FD3J0EfCNxJuIbg
-TwitterSecret=kOS4ZVvYWmlmqxyU1mtDYrYK3lAwu5cscXrLCq6qqpzbCoAkLi
-twitterUserName=wso2qa
-twitterPassword=wso2@qa
+TwitterClientId=WWdo748rnXiF8BkExp7ziVYNm
```

Comment:
Shall we remove the actual values and keep this as a template ex: TwitterClientId=<Twitter-ClientID-goes Hear>

Reply:
Currently, the test grid is using these values to execute the scenario, hence providing a template will not be a suitable option. If we do not pass these values in the properties file, we need to come up with a solution for this